Frameworks/CoreText/CTFont.mm

```diff
@@ -945,6 +945,12 @@ CFTypeID CTFontGetTypeID() {
     return __kCTFontTypeID;
 }
 
+// Private function for getting the XAML-compatible family name
+CFStringRef _CTFontGetCompatibleFamilyName(CTFontRef font) {
```

Comment:
Get rid of this and call the _DWrite version in UIKit. Or insert XAML between Get and Compatible :)